ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/DeduplicateTest.java

```diff
@@ -23,87 +24,78 @@
 import org.apache.beam.sdk.values.TypeDescriptors;
 import org.junit.Rule;
 import org.junit.Test;
-import redis.embedded.RedisServer;
 
 public class DeduplicateTest {
 
   @Rule
   public final transient TestPipeline pipeline = TestPipeline.create();
 
+  @Rule
+  public final RedisServer redis = new RedisServer();
+
   @Test
   public void testOutput() throws IOException {
     DecoderOptions decoderOptions = pipeline.getOptions().as(DecoderOptions.class);
     decoderOptions.setDeduplicateExpireDuration(StaticValueProvider.of("24h"));
+    decoderOptions.setRedisUri(StaticValueProvider.of(redis.uri));
     Parsed parsedOptions = DecoderOptions.parseDecoderOptions(decoderOptions);
     Integer ttlSeconds = parsedOptions.getDeduplicateExpireSeconds().get();
-
-    int redisPort = new redis.embedded.ports.EphemeralPortProvider().next();
-    // create testing redis server
-    final RedisServer redis = RedisServer.builder().port(redisPort).setting("bind 127.0.0.1")
-        .build();
-    final URI redisUri = URI.create("redis://localhost:" + redisPort);
-    redis.start();
-
-    try {
```

Comment:
Changes in this block are just indentation because we no longer need the try/catch with the RedisServer rule